Plots/ColormapComparisons/CB_Temperature.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+CB_Temperature.py
+================
+
+This script illustrates the following concepts:
+   - Drawing filled contours over a Mercator map
+   - Setting the spacing for latitude/longitude grid lines
+   - Turning off the map perimeter (boundary)
+   - Making the plot larger using viewport resources
+   - Turning off map fill
+   - Spanning part of a color map for contour fill
+   - Using 'inferno' color scheme instead of 'rainbow' to follow best practices for visualizations 
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/proj_2.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/proj_2_lg.png
+"""
+
+
+import numpy as np
+import xarray as xr
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+from geocat.viz import util as gvutil
+
```

Comment:
I think this file could benefit from being broken up into chucks using that lines of hashes. That way Sphinx will show code blocks versus one long file.